### DISCUSSION TITLE
Infinite Loading Emulator Android

### DISCUSSION THREAD

**Discussion by josuesalomao** [beeware/briefcase#1765] [Q&A]
### Describe the bug

When I run it on an Android emulator, it doesn't stop loading. The program keeps loading infinitely. Why doesn't anyone talk about this bug? I have checked my computer and several other things, but it is really a problem when running to use the android emulator.

### Expected behavior

check this bug

### Environment

- Operating System:
- Python version:
- Software versions:
  - Briefcase:
  - Toga:
  - ...

### Logs

```

```

**Comment by freakboy3742** [comment]
I've moved this issue to a discussion on the Briefcase project; you opened it as a bug on the project website.

It's very difficult to diagnose what is going on here, because you haven't provided any details other than "it doesn't work" - we don't even know what operating system you're using (despite an explicit prompt to provide that information in the bug submission form).

If you're on Windows, it's possible you've found #1573. We believe we've fixed this bug in the development version of Briefcase; the next release (due in a week or so) will include the fix. Ticket #1573 includes some more details, including a possible workaround.

If that's not the problem you're seeing, then we'll need more details - specifically, details about the platform you're using, and the log file that Briefcase generates when it fails. If Briefcase isn't generating a log file, you can pass `--log` to any Briefcase command to generate a log file.
